modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated/3d/3d-segregated-velocity.i

```diff
@@ -0,0 +1,223 @@
+mu = 0.002
+rho = 1.0
+advected_interp_method = 'average'
+velocity_interp_method = 'rc'
+
+pressure_tag = "pressure_grad"
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 3
+    dx = '0.2'
+    dy = '0.2'
+    dz = '0.8'
+    ix = '3'
+    iy = '3'
+    iz = '12'
+  []
+[]
+
+[GlobalParams]
+  rhie_chow_user_object = 'rc'
+[]
+
+[Problem]
+  nl_sys_names = 'u_system v_system w_system pressure_system'
+  previous_nl_solution_required = true
+  error_on_jacobian_nonzero_reallocation = true
+[]
+
+[UserObjects]
+  [rc]
+    type = INSFVRhieChowInterpolatorSegregated
+    u = vel_x
+    v = vel_y
+    w = vel_z
+    pressure = pressure
+  []
+[]
+
+[Variables]
+  [vel_x]
+    type = INSFVVelocityVariable
+    initial_condition = 0.0
+    nl_sys = u_system
+    two_term_boundary_expansion = false
```

Comment:
why is this disabled everywhere? what s the concern?

should this be enforced programmatically with an error in the Rc object?